uber/server.py

```diff
@@ -32,6 +32,7 @@ def magfest_js(self):
         js_consts = json.dumps({k: v for k, v in consts.items() if isinstance(v, (bool, int, str))}, indent=4)
         return '\n'.join([
             'angular.module("magfest", [])',
+            '.constant("c", {})'.format(js_consts),
```

Comment:
Our Angular code should probably just use the same ``c`` name that we're using in our Python code, for the sake of consistency.  I left the old ``magconsts`` definition for backwards-compatibility, and we can remove it after there's no more code left which relies on it.